### PR TITLE
CI: K8sFQDNTest retry initial DNS lookups

### PR DESCRIPTION
**Pull request by raybejjani**
We target jenkins.cilium.io and vagrant-cache.ci.cilium.io as the remote
targets for the FQDN tests. Sometimes, the DNS lookups fail and so does
the test. We now retry for 30 seconds. When your DNS lookup at first
doesn't succeed, try try again.

### CODE REVIEW

**Comment by raybejjani** [comment]
test-focus K8sFQDNTest.*

**Comment by raybejjani** [comment]
test-gke K8sFQDNTest.*

File: test/k8sT/fqdn.go [hunk 1]
```diff
@@ -45,14 +46,27 @@ var _ = Describe("K8sFQDNTest", func() {
...
+
+			addrs, lookupErr = net.LookupHost("jenkins.cilium.io")
+			if lookupErr != nil {
+				lookupErr = fmt.Errorf("error looking up jenkins.cilium.io: %s", lookupErr)
+				return false
+			}
+			worldInvalidTargetIP = addrs[0]
+
+			return true
+		}, "Could not get vagrant-cache.ci.cilium.io and jenkins.cilium.io IPs", &helpers.TimeoutConfig{Timeout: 30 * time.Second})
```
Comment (nebril) [inline test/k8sT/fqdn.go]: Can we use `helpers.MidCommandTimeout`?

**Review by nebril — CHANGES_REQUESTED** [review]
One small change request, lgtm otherwise

**Comment by raybejjani** [comment]
test-gke K8sFQDNTest.*

**Comment by raybejjani** [comment]
test-focus K8sFQDNTest.*

**Comment by nebril** [comment]
build fails because of `time` import

**Comment by raybejjani** [comment]
test-focus K8sFQDNTest.*

**Comment by raybejjani** [comment]
test-gke K8sFQDNTest.*

**Pull request merged**